user-guide/Troubleshooting/Known_issues/KI_Upgrade_fails_VerifyNoObsoleteAPIDeployed_prerequisite.md

```diff
@@ -0,0 +1,37 @@
+---
```

Comment:
Probably best to go over these tips: https://docs.dataminer.services/CONTRIBUTING.html#tips-for-writing-documentation. There are some things here that should be avoided according to those (avoid contractions).